azurelinuxagent/common/conf.py

```diff
@@ -80,6 +80,9 @@ def load_conf_from_file(conf_file_path, conf=__conf__):
         raise AgentConfigError(("Failed to load conf file:{0}, {1}"
                                 "").format(conf_file_path, err))
 
+def enable_rdma(conf=__conf__):
+    return conf.get_switch("OS.EnableRDMA", False)
```

Comment:
- Updated the README.md file to document OS.EnableRDMA
- I could not find any evidence of the existing (2.0 branch) variables thus I did not refer or comment on those
- I placed the method in somewhat alpha order (e before g) am happy to move it anywhere else in the file, just didn't see any intrinsic organization of method placement